.github/workflows/usage.yml

```diff
@@ -11,25 +10,26 @@ on: # run any PRs and main branch changes
   workflow_dispatch:
 
 jobs:
-
-  tortellini:
+  current-repository:
     runs-on: ubuntu-latest
-
     steps:
       - uses: actions/checkout@v2
```

Comment:
unrelated to this PR but just a thought: maybe we should make checking out the code part of tortellini. Something like, 
```
if (repositories==='') {
    find out which remote to use, of use GITHUB_* env vars
    git clone that remote
    proceed as normal
}
```

Reply:
Created https://github.com/tortellini-tools/action/issues/139